Org profile (#17)
puppetlabs/.github: profile/README.md

## Welcome to the Puppet community 👋

> ![Puppet folk having a grand time at Portland's annual Pride parade.](https://raw.githubusercontent.com/puppetlabs/.github/org-profile/profile/Community-Marquee-1792x768.png)

Reviewer: Looks like where you've put the image will end up at
```suggestion
![Puppet folk having a grand time at Portland's annual Pride parade.](https://raw.githubusercontent.com/puppetlabs/.github/profile/Community-Marquee-1792x768.png)
```

### 🎪 An interconnected community

You can get involved in the [Puppet community](https://puppet.com/community/) by connecting at events, getting or giving help in any of our many channels, sharing modules on the Forge, contributing code, docs, or any of your valuable insights--and many other ways, too! However you decide to jump in, we know you’ll learn a lot and enjoy meeting the smart, friendly people who make up the Puppet community. Maybe you'll even fall in with our good friends at [🦊 Vox Pupuli](https://voxpupuli.org)!

- 💬 https://slack.puppet.com
- 🎫 https://tickets.puppet.com
- 🎩 https://forge.puppet.com
- 🦊 https://voxpupuli.org
- 🎙 https://dev.to/puppet

Don't forget that all participation is governed by our [Code of conduct](https://pup.pt/conduct).

### 🎁 Contributing to the ecosystem

Community contributions are awesome and part of the way that we make sure that our tools continue to solve your problems. We want to keep it as easy as possible to participate and make contributions. There are a few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things, of course!

> Most of our projects will use our standard [contribution guidelines](https://github.com/puppetlabs/.github/blob/master/CONTRIBUTING.md). but you should check individual repositories for a more specific guide before submitting a PR.

Reviewer: ooh should we make the `master`->`main` switch for this repo?

Author: good call! I'll do that separately

> Most of our projects will use our standard [contribution guidelines](https://github.com/puppetlabs/.github/blob/master/CONTRIBUTING.md). but you should check individual repositories for a more specific guide before submitting a PR.

Reviewer: ```suggestion
Most of our projects will use our standard [contribution guidelines](https://github.com/puppetlabs/.github/blob/master/CONTRIBUTING.md), but you should check individual repositories for a more specific guide before submitting a PR.
```

### 💻 Wanna come work with us?

> Check out our 📜 [jobs listing](http://puppet.com/jobs) and see what strikes your fancy.

Reviewer: ¯\_(ツ)_/¯ this feels more natural to me
```suggestion
Check out our 📜 [job listings](http://puppet.com/jobs) and see what strikes your fancy.
```
